asv/benchmarks/cwt_benchmarks.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import pywt
+
+
+class CwtTimeSuiteBase(object):
+    """
+    Set-up for CWT timing.
+    """
+    params = ([32, 128, 512],
+              ['cmor', 'cgau4', 'fbsp', 'gaus4', 'mexh', 'morl', 'shan'],
+              [16, 64, 256])
+    param_names = ('n', 'wavelet', 'max_scale')
+
+    def setup(self, n, wavelet, max_scale):
+        try:
+            from pywt import cwt
+        except ImportError:
+            raise NotImplementedError("cwt not available")
```

Comment:
The difference seems to be that with my approach the test shows up as "n/a" instead of as "failed".  I used NotImplementedError in setup based on the following text in the ASV docs

> If setup raises a NotImplementedError, the benchmark is marked as skipped for the parameter values in question.

see for example the following output when running the CWT tests on 0.4.0 where it is not present:
(scroll right to see the n/a)

```
12:58 $ asv run --python=3.5 --quick --bench="Cwt*" v0.4.0^!
· No `environment_type` specified in asv.conf.json. This will be required in the future.
· Creating environments
· Discovering benchmarks
·· Uninstalling from conda-py3.5-Cython-numpy
·· Building for conda-py3.5-Cython-numpy.......
·· Installing into conda-py3.5-Cython-numpy..
· Running 1 total benchmarks (1 commits * 1 environments * 1 benchmarks)
[  0.00%] · For pywt commit hash 3f4f46a9:
[  0.00%] ·· Building for conda-py3.5-Cython-numpy.^[[1;2C....
[  0.00%] ·· Benchmarking conda-py3.5-Cython-numpy
[100.00%] ··· Running cwt_benchmarks.CwtTimeSuite.time_cwt                                                                                          n/a;...
```